manifests/pulp.pp

```diff
@@ -15,6 +15,7 @@
   Stdlib::Absolutepath $repo_export_dir = $::katello::repo_export_dir,
   String $repo_export_dir_owner = $::katello::user,
   String $repo_export_dir_group = $::katello::group,
+  Boolean $manage_httpd = $::katello::pulp_manage_httpd,
```

Comment:
The reasoning behind this parameter is that we have 3 ways of deploying pulp:
* Deploy as part of the foreman vhost (puppet-katello)
* Deploy as part of the devel vhost (puppet-katello_devel)
* Deploy a full vhost (puppet-foreman_proxy_content)

They are all almost identical and I'd like a solution where we can reuse. At least katello & katello_devel should be doable.